core/src/task.cpp

```diff
@@ -292,7 +292,6 @@ void Task::compute() {
 
 bool Task::plan(size_t max_solutions) {
 	auto impl = pimpl();
-	reset();
```

Comment:
Out of curiosity: What should `{ plan(1); plan(1); }` do? Are you expecting it to find more solutions?
The stages are still (re)initialized...
The ROS message you are mentioning in the log message is required to indicate to rviz that this task started over.